### PR TITLE
{math}[foss/2023b,gfbf/2023b] gap v4.13.0, Normaliz v3.10.3

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/g/gap/gap-4.13.0-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ('GMP', '6.3.0'),
+    ('libreadline', '8.2'),
+    ('zlib', '1.2.13'),
+    ('4ti2', '1.6.10'),  # needed by 4ti2Interface, HeLP
+    ('cddlib', '0.94m'),  # needed by CddInterface
+    ('cURL', '8.3.0'),  # needed by curlInterface
+    ('lrslib', '7.2'),  # needed by HeLP
+    ('ncurses', '6.4'),  # needed by Browse
+    ('Normaliz', '3.10.3'),  # needed by NormalizInterface, HeLP
+    # ('Singular', '4.3.2p16'),  # needed by singular
```
Comment (boegel) [inline easybuild/easyconfigs/g/gap/gap-4.13.0-foss-2023b.eb:33]: @PetrKralCZ Is this no longer needed?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/gap/gap-4.13.0-foss-2023b.eb (hunk 1 above)
Comment (PetrKralCZ) [reply easybuild/easyconfigs/g/gap/gap-4.13.0-foss-2023b.eb]: It is an optional dependency but I guess there was a reason it was included in older versions of easyconfigs for `gap` and I don't really remember why I commented it out so I am putting it back.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/c93742725e77c1db6829023bc079c3f9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**